alembic/env.py

```diff
@@ -0,0 +1,78 @@
+from logging.config import fileConfig
+
+from sqlalchemy import engine_from_config
+from sqlalchemy import pool
+
+from alembic import context
+
+# this is the Alembic Config object, which provides
+# access to the values within the .ini file in use.
+config = context.config
+
+# Interpret the config file for Python logging.
+# This line sets up loggers basically.
+if config.config_file_name is not None:
+    fileConfig(config.config_file_name)
+
+# add your model's MetaData object here
+# for 'autogenerate' support
+# from myapp import mymodel
+# target_metadata = mymodel.Base.metadata
+target_metadata = None
```

Comment:
The `target_metadata` is set to `None`. If you intend to use Alembic's autogenerate feature, you should assign it to your model's metadata object. For example, if you have a model `mymodel` with a `Base` class, you should uncomment and modify the lines to import your model and set `target_metadata = mymodel.Base.metadata`.